### PR TITLE
Fix eth_storageAt

### PR DESCRIPTION
**Pull request by philipglazman**
RPC returns preimage values for storage slots.

Used slither to test deployed contracts from tx generator. 

```
ERC20:
+-------------+-------------------------------------------------+------+------+--------+-------------------------------+
| name        |                                     type_string | slot | size | offset |                         value |
+-------------+-------------------------------------------------+------+------+--------+-------------------------------+
| nonces      |                     mapping(address => uint256) |    5 |  256 |      0 |                             0 |
| allowance   | mapping(address => mapping(address => uint256)) |    4 |  256 |      0 |                             0 |
| balanceOf   |                     mapping(address => uint256) |    3 |  256 |      0 |                             0 |
| totalSupply |                                         uint256 |    2 |  256 |      0 | 10000000000001000000000000000 |
| symbol      |                                          string |    1 |  256 |      0 |                         QUOTE |
| name        |                                          string |    0 |  256 |      0 |                         QUOTE |
+-------------+-------------------------------------------------+------+------+--------+-------------------------------+

OrderbookFactory:
+--------------------------+------------------------------------------+------+------+--------+--------------------------------------------+
| name                     |                              type_string | slot | size | offset |                                      value |
+--------------------------+------------------------------------------+------+------+--------+--------------------------------------------+
| activeMarkets            |                                address[] |    2 |  256 |      0 |                                          0 |
| variable                 |                                  uint256 |    1 |  256 |      0 |                                         21 |
| _owner                   |                                  address |    0 |  160 |      0 | 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 |
| _initializing            |                                     bool |   17 |    8 |    168 |                                      False |
| _initialized             |                                    uint8 |   17 |    8 |    160 |                                          0 |
| feeAddress               |                                  address |   17 |  160 |      0 | 0x0000000000000000000000000000000000000000 |
| takerFee                 |                                  uint256 |   16 |  256 |      0 |                                          0 |
| makerFee                 |                                   int256 |   15 |  256 |      0 |                                          0 |
| orderBooks               |       mapping(bool => Storage.OrderBook) |   14 |  256 |      0 |                                      False |
| depositedAmountPrecision |                                  uint256 |   13 |  256 |      0 |                                          0 |
| quotePrecision           |                                  uint256 |   12 |  256 |      0 |                                          0 |
| basePrecision            |                                  uint256 |   11 |  256 |      0 |                                          0 |
| minQuantityIncrement     |                                  uint256 |   10 |  256 |      0 |                                          0 |
| tokenBalance             | mapping(uint48 => Storage.TokenBalances) |    9 |  256 |      0 |                                          0 |
| quoteToken               |                                  address |    8 |  160 |      0 | 0x0000000000000000000000000000000000000000 |
| baseToken                |                                  address |    7 |  160 |      0 | 0x0000000000000000000000000000000000000000 |
| registeredAddresses      |               mapping(uint48 => address) |    6 |  256 |      0 |                                          0 |
| registeredIds            |               mapping(address => uint48) |    5 |  256 |      0 |                                          0 |
| registrationCounter      |                                   uint48 |    4 |   48 |     40 |                                          0 |
| idCounter                |                                   uint40 |    4 |   40 |      0 |                                          0 |
| minPriceIncrement        |                                  uint256 |    3 |  256 |      0 |                                          0 |
| maxPrice                 |                                  uint256 |    2 |  256 |      0 |                                          0 |
| minPrice                 |                                  uint256 |    1 |  256 |      0 |                                         21 |
| admin                    |                                  address |    0 |  160 |      0 | 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 |
+--------------------------+------------------------------------------+------+------+--------+--------------------------------------------+
```

### CODE REVIEW

File: monad-triedb-utils/src/lib.rs [hunk 1]
```diff
@@ -155,17 +155,23 @@ impl TriedbEnv {
...
-                    let _ = send.send(TriedbResult::EncodingError);
-                    return;
-                }
-                let mut storage_value = [0_u8; 32];
-                for (byte, storage) in result.into_iter().rev().zip(storage_value.iter_mut().rev())
-                {
-                    *storage = byte;
-                }
-                let _ = send.send(TriedbResult::Storage(storage_value));
+                let _ = match U256::decode(&mut result.as_slice()) {
```
Comment (eerkaijun) [inline monad-triedb-utils/src/lib.rs:158]: what does the `decode` do here? is it rlp decoding? if so, is there a change in schema in triedb? the last time i check the storage slot value in triedb is not rlp encoded

Comment (philipglazman) [reply monad-triedb-utils/src/lib.rs:158]: I forgot to update the execution submodule commit. The format of the storage leaf is now rlp encoded https://github.com/monad-crypto/monad/pull/744

**Comment by philipglazman** [comment]
> what's the motivation for this PR? with this change it seems like the returned value from RPC differs from ethereum RPC

What is the difference? Ran against anvil is the RPC response is now the same.

File: monad-triedb-utils/src/lib.rs (hunk 1 above)
Comment (eerkaijun) [reply monad-triedb-utils/src/lib.rs:158]: got it, since the schema changed, can you also update the readme? https://github.com/monad-crypto/monad-bft/blob/master/monad-triedb/README.md

File: monad-blockdb-utils/Cargo.toml [hunk 2]
```diff
@@ -11,4 +11,4 @@ monad-blockdb = { path = "../monad-blockdb" }
 
 heed = { workspace = true }
 rayon = { workspace = true }
-tokio = { workspace = true }
```
Comment (ariqchowdhury) [inline monad-blockdb-utils/Cargo.toml:14]: is this an unrelated change?

Comment (omegablitz) [reply monad-blockdb-utils/Cargo.toml:14]: I think we forgot to include that feature in `monad-blockdb-utils`, even though we're using tokio::sync inside. So it'll compile fine if you compile a crate that depends on `monad-blockdb-utils` that includes the `sync` feature transitively, but it won't build if you `cargo check -p monad-blockdb-utils`.

File: monad-triedb-utils/src/lib.rs (hunk 1 above)
Comment (philipglazman) [reply monad-triedb-utils/src/lib.rs:158]: done

File: monad-blockdb-utils/Cargo.toml (hunk 2 above)
Comment (philipglazman) [reply monad-blockdb-utils/Cargo.toml:14]: Correct, I originally forgot to include that feature
